internal/server/iigointernal/judiciary.go

```diff
@@ -132,3 +151,140 @@ func softMergeSanctionThresholds(clientSanctionMap map[roles.IIGOSanctionTier]ro
 	}
 	return defaultMap
 }
+
+// getIslandSanctionTier if statement based evaluator for which sanction tier a particular score is in
+func getIslandSanctionTier(islandScore roles.IIGOSanctionScore, scoreMap map[roles.IIGOSanctionTier]roles.IIGOSanctionScore) roles.IIGOSanctionTier {
```

Comment:
this assumes that the sanction costs scale linearly. if sanction tier 4 is less than 3 this would not function as expected. However i assume you dont want that case to ever happen, so it might be worth adding a check for this when the judge returns their scoreMap

Reply:
Sure, I might end up adding a check when the clients give us the map in the first place.

Reply:
Done